src/templates/vault_transit.hcl.j2

```diff
@@ -0,0 +1,10 @@
+seal "transit" {
+  address         = "{{ vault_addr }}"
+  disable_renewal = "false"
+  key_name        = "{{ key_name }}"
+  mount_path      = "{{ mount_path }}"
+  token           = "{{ vault_token }}"
+  {% if tls_ca_cert %}
```

Comment:
Are there cases where we won't have a ca cert for this?

Reply:
No, not anymore. This can be removed.

Reply:
Fixed in https://github.com/canonical/vault-k8s-operator/pull/365/commits/0a90837bbcdef6e42799e4e45129a758b899699f